code/modules/food&drinks/food/snacks_bread.dm

```diff
@@ -7,6 +7,7 @@
 	bitesize = 2
 	custom_food_type = /obj/item/weapon/reagent_containers/food/snacks/customizable/sandwich
 	filling_color = "#FFA500"
+	slot_flags = SLOT_HEAD // i'm going to kill myself. i'm going to murder my own self.
```

Comment:
Me:
>>>Still don't know why it isn't ```SLOT_MASK```.

>>Anyone?

>Anyone explain? I mean cigarettes and cigars aren't ```SLOT_HEAD```.
Unless this actually just makes you put a piece of toast on your HEAD like a hat and not in your mouth.
In which case: What the hell?

>Just look at how cigarettes are, how hard can that be?
EDIT: Ah, [cigarettes are a child of masks](https://github.com/tgstation/-tg-station/blob/85ff51b3f7e7ad0922eb49ee58963b54f7342e36/code/game/objects/items/weapons/cigs_lighters.dm#L96). So it may need some code/vars from [mask](https://github.com/tgstation/-tg-station/blob/master/code/modules/clothing/clothing.dm#L81) and [clothing](https://github.com/tgstation/-tg-station/blob/master/code/modules/clothing/clothing.dm#L1).

Chocobro:
>it's not even fixed

An0n3:
>gay anime breadhead bullshit
>cant even put it in the mouth
>have to wear your bread like a hat holy shit